runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCachePartition.java

```diff
@@ -360,14 +364,14 @@ public void writeEntry(
         ConverterHandler convertKey,
         ConverterHandler convertValue,
         boolean verbose,
-        List<KafkaTopicHeaderType> headerTypes)
+        KafkaTopicTransformsConfig transforms)
```

Comment:
I think this is assuming we can specify either `extract-key` or `extract-headers` only once in the `transforms` array, right?